docs/website/binding_types_reference_guide.md

```diff
@@ -2083,6 +2083,52 @@ Example:
 You can then call the extension method `GetDomain` to get the domain constant of
 any error.
 
+
```

Comment:
that's inside the `#Enum` section - but the attribute is not (exclusively) related to `enum` support, like the other `##` subsections before and after your addition